platform/qt/resources/common.qrc

```diff
@@ -1,6 +1,6 @@
 <RCC>
     <qresource prefix="/">
-        <file alias="icon.png">../../../common/icon.png</file>
+        <file alias="icon.png">../../../common/mb-icon-blue-square.png</file>
```

Comment:
This is not shipped. Just a cosmetic on the Qt test app (`make run-qt-app`) so it has an icon at the taskbar.

![screenshot from 2017-04-20 15-30-27](https://cloud.githubusercontent.com/assets/996307/25230941/9b1ee08c-25de-11e7-8c32-640782fc3003.png)